### PR TITLE
Dev/rostik b/mks 21 create web api project

### PR DESCRIPTION
**Pull request by Rostik18**

### CODE REVIEW

**Comment by Magfiozy** [comment]
Why did you add entity, swagger and controller in this branch? We talked about a new branch for every feature.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (bvzayats) [inline .gitignore:1]: Improve your gitignore file

File: Andriy Burtso/about-me.txt [hunk 2]
```diff
@@ -1,13 +0,0 @@
-Рожевозаді поні так собі, вони мені не дуже подобаються. А от що до морозива то я люблю ТМ Пломбір з печеньками.
```
Comment (bvzayats) [inline Andriy Burtso/about-me.txt:1]: Don't remove your teammate folders and files!

File: Bohdan_Khamurda/Task_0.txt [hunk 3]
```diff
@@ -1,3 +0,0 @@
-���� �� ��������� - �� ������ �������.
```
Comment (bvzayats) [inline Bohdan_Khamurda/Task_0.txt:1]: Don't remove your teammate folders and files!

File: README.md [hunk 4]
```diff
@@ -1 +0,0 @@
-# nedo-net
```
Comment (bvzayats) [inline README.md:1]: Don't remove your teammate folders and files! And my files!

File: Serhii Nykon/description.txt [hunk 5]
```diff
@@ -1,9 +0,0 @@
-Flamingos are large birds that are identifiable by their long necks, sticklike legs and pink or reddish feathers. Flamingos embody the saying "you are what you eat." The pink and reddish colors of a flamingo's feathers come from eating pigments found in algae and invertebrates.
```
Comment (bvzayats) [inline Serhii Nykon/description.txt:1]: Don't remove your teammate folders and files!

File: Tetiana_Sulyma/Pony.txt [hunk 6]
```diff
@@ -1,9 +0,0 @@
-My Little Pony: Friendship is Magic is an animated television series created by Hasbro Studios (United States) and DHX Media (Canada), an idea based on the line of toys My Little Pony from Hasbro, Inc.
```
Comment (bvzayats) [inline Tetiana_Sulyma/Pony.txt:1]: Don't remove your teammate folders and files!

File: The quest/Task.txt [hunk 7]
```diff
@@ -1,34 +0,0 @@
-Вітаю, мої юні юзери гіта.
```
Comment (bvzayats) [inline The quest/Task.txt:1]: Don't remove your teammate folders and files!

File: Volodymyr_Tsisinskyi/archery.txt [hunk 8]
```diff
@@ -1,3 +0,0 @@
-Archery is cool.
-The unwitting desire to hit the target and extraordinary calm of mind will necessarily lead to new heights.
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/archery.txt:2]: Don't remove your teammate folders and files!

File: Rostik_Baitsar/Text.txt [hunk 9]
```diff
@@ -1,2 +0,0 @@
-����� ���.
```
Comment (bvzayats) [inline Rostik_Baitsar/Text.txt:1]: Don't remove your teammate folders and files! And Your old tasks files!

File: Serhii Nykon/.gitignore [hunk 10]
```diff
@@ -1,4 +0,0 @@
-.vs/
```
Comment (bvzayats) [inline Serhii Nykon/.gitignore:1]: Don't remove your teammate folders and files!

File: Serhii Nykon/NedoNet.API/Startup.cs [hunk 11]
```diff
@@ -1,21 +0,0 @@
-﻿using Microsoft.AspNetCore.Builder;
```
Comment (bvzayats) [inline Serhii Nykon/NedoNet.API/Startup.cs:1]: Don't remove your teammate folders and files!

File: Serhii Nykon/NedoNet.API/appsettings.json [hunk 12]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (bvzayats) [inline Serhii Nykon/NedoNet.API/appsettings.json:1]: Don't remove your teammate folders and files!

File: Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.csproj.CoreCompileInputs.cache [hunk 13]
```diff
@@ -0,0 +1 @@
+871f5f3644fde745b70d4220cb8c415209315bff
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.csproj.CoreCompileInputs.cache:1]: Improve your gitignore

File: Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.AssemblyInfoInputs.cache [hunk 14]
```diff
@@ -0,0 +1 @@
+a6bd044af87f89e06eef562b40bad94132385d39
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.AssemblyInfoInputs.cache:1]: Improve your gitignore

File: Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.RazorAssemblyInfo.cache [hunk 15]
```diff
@@ -0,0 +1 @@
+9ef567e268bcedc1f82869c5dc7d2e52ce4fca0c
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/obj/Debug/netcoreapp2.2/Festival.RazorAssemblyInfo.cache:1]: Improve your gitignore

File: Rostik_Baitsar/Festival/Festival/Startup.cs [hunk 16]
```diff
@@ -0,0 +1,72 @@
+﻿using System;
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Startup.cs]: Remove unused namespaces

File: Rostik_Baitsar/Festival/Festival/Program.cs [hunk 17]
```diff
@@ -1,7 +1,14 @@
-﻿using Microsoft.AspNetCore;
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Program.cs:1]: Remove unused namespaces

File: Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs [hunk 18]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs]: Remove unused namespaces

File: Rostik_Baitsar/Festival/Festival/Models/BandContext.cs [hunk 19]
```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Models/BandContext.cs:1]: Remove unused namespaces in the WHOLE system.

**Review by bvzayats — COMMENTED** [review]
This PR will not be merged. Please check all the comments. Fix all troubles and mistakes. Split different functional by different branches. And make another PRs. Please notify me that you saw these comments, noticed them yourself and I'll remove this PR.

**Pull request closed without merging**